helpers/sortByMapping.js

```diff
@@ -9,32 +9,48 @@ module.exports = function( dust ) {
   * @description Sorting method that orders a list of things based on a provided map
   * @param {object} eventsObject Object to sort
   * @param {string} map Map to use to sort with
-  * @example {@_sortByMapping eventsObject=eventsReply.events map="ASHLKE, ASHWKE, ASHJBE} {/_sortByMapping} output loop of events
+  * @param {string} sortKeys - structure of response to sort on
```

Comment:
i would suggest calling this something like `indexes` as they are the indexes to the `sortObject`